nf_core/modules/list.py

```diff
@@ -93,7 +93,7 @@ def pattern_msg(keywords):
                     repo_modules = repo_entry.get("modules")
                     if repo_modules is None:
                         raise UserWarning(
-                            "You 'modules.json' file is not up to date. Please remove it and rerun the command"
+                            "You 'modules.json' file is not up to date. You can fix it by running 'nf-core modules update'."
```

Comment:
done. the function is now called `has_git_url_and_modules` (in preparation for the removal of base_path). Because we call it in `check_up_to_date` a few lines above this, I could remove this whole if clause.